src/main/java/service/StorageService.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+import java.util.List;
+import java.util.Map;
+import model.FruitRecord;
+
+public interface StorageService {
+    Map<String, Integer> processingData(List<FruitRecord> fruitRecordList);
```

Comment:
I don't understand why we need this class and what do this method. Let's name all you're classes and methods laconically and more clearly.